src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -275,6 +276,17 @@ bool clang_c_convertert::get_decl(const clang::Decl &decl, exprt &new_expr)
   case clang::Decl::Typedef:
     break;
 
+  case clang::Decl::BuiltinTemplate:
+  {
+    // expanded by clang itself
+    const clang::BuiltinTemplateDecl &btd =
+      static_cast<const clang::BuiltinTemplateDecl &>(decl);
+    assert(
```

Comment:
An error message followed by an abort would be better here. This assert will not be triggered for release builds.

Reply:
Fixed.